toolchain/sem_ir/inst_namer.cpp

```diff
@@ -549,11 +652,30 @@ auto InstNamer::CollectNamesInBlock(ScopeId scope_id,
         CollectNamesInBlock(interface_scope_id, inst.decl_block_id);
         continue;
       }
+      case InterfaceWitness::Kind: {
+        // TODO: Include name of interface.
+        add_inst_name("interface");
+        continue;
+      }
+      case CARBON_KIND(InterfaceWitnessAccess inst): {
+        std::string name;
+        llvm::raw_string_ostream out(name);
+        out << "elt" << inst.index.index;
```

Comment:
We use "elt" in a couple of other places. Is there a more specific name we could give this?

Reply:
I agree it's a little mysterious from the value name alone, but I think from the instruction name `interface_witness_access` it's reasonably clear what it means. Maybe we'll find a better name later but this seems good enough for now.

Reply:
Done.